src/astra_driver.cpp

```diff
@@ -54,6 +54,8 @@ AstraDriver::AstraDriver(rclcpp::node::Node::SharedPtr& n, rclcpp::node::Node::S
     pnh_(pnh),
     device_manager_(AstraDeviceManager::getSingelton()),
     config_init_(false),
+    depth_frame_id_("openni_depth_optical_frame"),
+    color_frame_id_("openni_color_optical_frame"),
```

Comment:
This should appear in the same order as they are declared in the interface. Otherwise this produces warnings:
```
warning: ‘astra_wrapper::AstraDriver::depth_frame_id_’ will be initialized after [-Wreorder]
   std::string depth_frame_id_ ;
```